authui/src/build-authflowv2.html

```diff
@@ -4,6 +4,7 @@
     <script type="module" src="authflowv2.ts"></script>
     <script src="colorscheme.ts"></script>
     <link rel="stylesheet" href="authflowv2/index.css" />
+    <link rel="stylesheet" href="npm:cropperjs/dist/cropper.min.css" />
```

Comment:
Questions: Should this be added here or in __html_head template?

Reply:
Should be here as discussed with louis